data/skipper-entrypoint.sh

```diff
@@ -8,5 +8,10 @@ fi
 groupadd -g ${SKIPPER_DOCKER_GID} --non-unique docker
 usermod -G root,docker ${SKIPPER_USERNAME}
 
-su -m ${SKIPPER_USERNAME} -c "$@"
 
+while true; do
+	su -m ${SKIPPER_USERNAME} -c "$@"
+	if [ x$? == x0 ]; then
+	    break
+	fi
+done
```

Comment:
arghhhh.... this will affect all commands running inside a container (not only bash).
consider a command that takes a long time to run. now assume the user wants to kill this long command by sending Ctrl-C. the result will be running the command again instead of killing it.